app/features/AccountSlice.ts

```diff
@@ -473,14 +473,22 @@ async function updateAccountFromAccountInfo(
 
     if (validatorId !== undefined) {
         const poolStatus = await getPoolStatusLatest(validatorId);
-        if (poolStatus.isSuspended) {
+        /* if (poolStatus.isSuspended) {
             dispatch(
                 setSuspensionStatus({
                     address: account.address,
                     isSuspended: true,
                 })
             );
         }
+            */
```

Comment:
If we don't need this, can we remove it instead? If it's needed in future, it would be good with a comment explaining why 🙂